src/shared/components/NewApplication/FileDropzone.tsx

```
@@ -0,0 +1,128 @@
+
+        if (maxSizeMB > 0) {
+            const limit = maxSizeMB * 1024 * 1024;
+            filtered = filtered.filter((f) => f.size <= limit);
+        }
+
+        try {
+            const dt = new DataTransfer();
+            filtered.forEach((f) => dt.items.add(f));
+            onFiles(dt.files);
+            return;
+        } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)